detection_rules/main.py

```diff
@@ -119,21 +119,19 @@ def toml_lint(rule_file):
     """Cleanup files with some simple toml formatting."""
     if rule_file:
         contents = pytoml.load(rule_file)
-        rule = Rule(path=rule_file.name, contents=contents)
-
-        # removed unneeded defaults
-        for field in rule_loader.find_unneeded_defaults_from_rule(rule):
-            rule.contents.pop(field, None)
-
-        rule.save(as_rule=True)
+        rules = [TOMLRule(path=rule_file.name, contents=contents)]
```

Comment:
doesn't this need to set with `TOMLRuleContents` ?

Reply:
yeah. I accidentally blew away this change and had to redo it, so I think we're looking at me missing it the second time

Reply:
```suggestion
        rules = [TOMLRule(path=rule_file.name, contents= TOMLRuleContents.from_dict(contents))]
```